doc/radosgw/notifications.rst

```diff
@@ -94,7 +96,10 @@ Request parameters:
 
 - Kafka endpoint 
 
- - URI: ``kafka://<fqdn>[:<port]``
+ - URI: ``kafka://[<user>:<password>@]<fqdn>[:<port]``
+ - if ``ca-location`` is provided, secure connection will be used for connection with the broker
```

Comment:
I'm still slightly confused--are the different endpoint security options being globbed together here?

Reply:
> I'm still slightly confused--are the different endpoint security options being globbed together here?

yes :-( will separate the options

Reply:
done